tsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
     "compilerOptions": {
-        "target": "ES2017",
-        "module": "esNext",
-        "moduleResolution": "node",
+        "lib": ["DOM", "ESNext"],
+        "target": "ESNext",
```

Comment:
Just out of curiosity: Why was this change required?

Reply:
TS didn't like `Promise.finally`.